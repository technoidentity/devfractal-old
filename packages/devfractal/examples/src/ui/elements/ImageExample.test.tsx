import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Image } from './ImageExample'

test('Image Example', () => {
  const component: ReactTestRenderer = renderer.create(<Image />)
  expect(component.toJSON()).toMatchSnapshot()
})
