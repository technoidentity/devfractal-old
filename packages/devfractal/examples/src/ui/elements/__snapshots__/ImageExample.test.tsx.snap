// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Normal Image
    </h1>
    <figure
      className="image is-128x128"
    >
      <img
        className=""
        src="https://bulma.io/images/placeholders/128x128.png"
      />
    </figure>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Fixed square images
    </h1>
    <section
      className="section"
    >
      <figure
        className="image is-16x16"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/128x128.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-24x24"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/128x128.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-32x32"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/128x128.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-48x48"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/128x128.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-64x64"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/128x128.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-128x128"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/128x128.png"
        />
      </figure>
    </section>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Rounded image
    </h1>
    <figure
      className="image is-96x96"
    >
      <img
        className="is-rounded"
        src="https://bulma.io/images/placeholders/128x128.png"
      />
    </figure>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Retina image
    </h1>
    <figure
      className="image is-128x128"
    >
      <img
        className=""
        src="https://bulma.io/images/placeholders/256x256.png"
      />
    </figure>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Responsive images with ratios
    </h1>
    <section
      className="section"
    >
      <figure
        className="image is-1by1"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/480x480.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-1by2"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/320x640.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-1by3"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/240x720.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-2by1"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/640x320.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-3by1"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/720x240.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-3by2"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/480x320.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-2by3"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/320x480.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-3by4"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/480x640.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-3by5"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/480x800.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-4by3"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/640x480.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-4by5"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/480x600.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-square"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/480x480.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-5by3"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/800x480.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-5by4"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/600x480.png"
        />
      </figure>
    </section>
    <section
      className="section"
    >
      <figure
        className="image is-9by16"
      >
        <img
          className=""
          src="https://bulma.io/images/placeholders/360x640.png"
        />
      </figure>
    </section>
  </section>,
]
`;
